mpsl/include/mpsl_fem_protocol_api.h

```diff
@@ -0,0 +1,281 @@
+ */
+int32_t mpsl_fem_lna_configuration_set(const mpsl_fem_event_t * const p_activate_event,
+                                       const mpsl_fem_event_t * const p_deactivate_event);
+
+/** @brief Clears up the configuration provided by the @ref mpsl_fem_lna_configuration_set function.
+ *
+ *
+ * @retval   0             LNA activate setup purge is successful.
+ * @retval   -NRF_EPERM    LNA is currently disabled.
+ */
+int32_t mpsl_fem_lna_configuration_clear(void);
+
+/** @brief Deactivates PA/LNA with immediate effect.
+ *
+ *  Deactivates PA/LNA with immediate effect - contrary to
+ *  @ref mpsl_fem_lna_configuration_clear or @ref mpsl_fem_pa_configuration_clear,
+ *  which both just set up the infrastructure for events which shall disable the PA/LNA.
+ *
+ * @param[in] type Whether to deactivate imeediately PA, LNA, or both (see @ref mpsl_fem_functionality_t).
+ */
+void mpsl_fem_deactivate_now(mpsl_fem_functionality_t type);
+
+/** @brief Instruct Front End Module to disable PA and LNA as soon as possible
+ *  using the group following the event.
+ *
+ * @param[in] event       Address of the event which is triggered when the abort condition occurs.
+ * @param[in] group       PPI Group which shall be disabled when the abort event is triggered.
+ *
+ * @retval   0            Setting of the abort sequence path is successful.
+ * @retval   -NRF_EPERM   Setting of the abort sequence path could not be performed.
+ */
+int32_t mpsl_fem_abort_set(uint32_t event, uint32_t group);
+
+/** @brief Adds one more PPI channel to the PPI Group prepared by the
+ *  @ref mpsl_fem_abort_set function.
+ *
+ * @param[in] channel_to_add PPI channel to add to the PPI group.
+ * @param[in] group          The said PPI group.
+ *
+ * @retval    0              Setting of the abort sequence path is successful.
+ * @retval    -NRF_EPERM     Setting of the abort sequence path could not be performed.
+ */
+int32_t mpsl_fem_abort_extend(uint32_t channel_to_add, uint32_t group);
+
+/** @brief Removes one PPI channel from the PPI Group prepared by the
+ *  @ref mpsl_fem_abort_set function.
+ *
+ * @param[in] channel_to_remove PPI channel to remove from the PPI group.
+ * @param[in] group             The said PPI group.
+ *
+ * @retval   0                  Setting of the abort sequence path is successful.
+ * @retval   -NRF_EPERM         Setting of the abort sequence path could not be performed.
+ */
+int32_t mpsl_fem_abort_reduce(uint32_t channel_to_remove, uint32_t group);
+
+/** @brief Clears up the configuration provided by the @ref mpsl_fem_abort_set
+ *  function.
+ *
+ * @retval   0          Clearing of the abort sequence path is successful.
+ * @retval   -NRF_EPERM Clearing was not done - the possible reason is that there was nothing to clear.
+ */
+int32_t mpsl_fem_abort_clear(void);
+
+/** @brief Cleans up the configured PA/LNA timer/radio instance and corresponding
+ *  allocated hardware resources. The function resets the hardware that has been
```

Comment:
When using `@brief` then there should be one sentence, and then a blank ` *` line before the more detailed description.

```
/** @brief The brief sentence description
 *
 * The longer sentences describing the features.
 *
 * @retval 0 Success
 */